packages/web/src/components/gcds-icon/gcds-icon.css

```diff
@@ -136,29 +136,81 @@
     margin-inline-end: var(--gcds-icon-margin-1000);
   }
 
+  /* Padding */
+  &.padding-fixed-width {
+    text-align: center;
+  }
+
+  &.size-caption.padding-fixed-width {
+    width: calc(var(--gcds-icon-fixed-width-caption) * 1em);
```

Comment:
We might have to go a bit bigger than 1em.
When I tested the fixed width with the cloud icon, the icon was bigger than the width:

![Screenshot 2023-02-06 at 11 13 18 AM](https://user-images.githubusercontent.com/8675814/217063680-f00aa08c-97fd-428f-bbcb-48420b620f5d.png)

Reply:
I think this is on purpose from font-awesome. The cloud might be bigger for optical reasons. Similar to baseline overshoot in typography: 
![image](https://user-images.githubusercontent.com/5230720/217267651-1ff52887-a635-4410-aa28-1bd6e306690b.png)

Reply:
Yeah could be, ok I'll resolve the comment for now. Maybe some we can research.